skietbaan_fe/src/components/RegisterStyles.css

```diff
@@ -9,8 +9,14 @@
   }
 }
 
-body{
-  background-image: url("../components/assets/Background.png");
+.Page-content{
+  width: auto;
+  height: stretch;
+  background-color: #504f51;
+  width:100%;
```

Comment:
Two width for the same class. Choose one

Reply:
didnt see that sorry, fixed it